src/components/material/MaterialDescription.tsx

```diff
@@ -17,20 +17,30 @@ export interface MaterialDescriptionProps {
 const MaterialDescription: React.FC<MaterialDescriptionProps> = ({ work }) => {
   const t = useText();
   const { searchUrl, materialUrl } = useUrls();
-  const inSeries = work.series;
-  const seriesMembersList = work.seriesMembers.map((item) => {
+  const { fictionNonfiction, series, subjects, seriesMembers, manifestations } =
+    work;
+
+  const seriesMembersList = seriesMembers.map((item) => {
     return {
       url: constructMaterialUrl(materialUrl, item.workId as WorkId),
-      term: item.titles.main[0]
+      term: item.titles.main.join(" / ")
     };
   });
-  const subjectsList = work.subjects.all.map((item) => {
+
+  const subjectsList = subjects.all.map((item) => {
     return {
       url: constructSearchUrl(searchUrl, item.display),
       term: item.display
     };
   });
-  const { fictionNonfiction } = work;
+
+  const filmAdaptationsList =
+    manifestations.bestRepresentation?.ownerWork.titles.main.map((item) => {
```

Comment:
Please reconsider this.

The original request says: Look in `relations.hasAdaptation` and get the title from the owner work of one of these manifestations - if available. That is not what has been implemented here.

As I see it this implementation takes the title(s) of the owner work of any best representation and prints that.